src/Build/mkdocs.jl

```diff
@@ -0,0 +1,94 @@
+function makemkdocs(build)
+    site_name = "Docile.jl"
+    repo_url = "https://github.com/MichaelHatherly/Docile.jl"
+    site_description = "Julia package documentation system."
+    site_author = "Michael Hatherly"
+    theme = "readthedocs"
+    docs_dir = joinpath(splitdir(Base.source_dir())[end], build)
```

Comment:
these seems like a reasonable api (or just a dict). Are any of these vars available elsewhere or should we just raise if they're not provided.

As mentioned in comments, one issue here is the ordering: alphabetical. Is, most of the time, not what you want. :s